README.md

```diff
@@ -90,11 +95,14 @@ pushed to a configured git repository.
 To run as a dev:
 
 1. Clone from git
-2. Run `poetry install -E airflow`
-4. Run `npm install from windmill/http/app`
-4. Run `windmill-dev start-frontend`
-3. Run `windmill-dev start-backend`
-5. Open `127.0.0.1:1234`
+2. Run `curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python`
+3. Run `source $HOME/.poetry/env`
+4. Activate your python virtualenv of choice
```

Comment:
This shouldn't be necessary - poetry should manage your virtual env once you pip install